site-service/src/main/resources/bootstrap-local.yml

```diff
@@ -0,0 +1,6 @@
+spring:
+  cloud:
+    config:
+      enabled: false
+      discovery:
+        enabled: false
```

Comment:
same comment - do we need to set this if the config is disabled anyway?

Reply:
Removed